src/pages/Profile/Tab/Tabs.jsx

```diff
@@ -0,0 +1,47 @@
+import { Wrapper, TabHead, EachTab,Outlet } from "./tabComponent";
+import {TabContent} from "./TabContent"
+import { useState } from "react";
+import { Post } from "../../../components";
+export const Tabs = () => {
+  const [activeTab, setActiveTab] = useState("tab1");
+  const currentStyle = {fontWeight: "700",color:"#ca2535",borderBottom:"3px solid #ca2535"}
```

Comment:
one suggestion from my side would be to move `currentStyle` outside the component because it is a constant value and does not change on every render.

Reply:
To escape from multiple import and export I haven't moved it to other const file as its just one line of code